Formula/fstrm.rb

```diff
@@ -0,0 +1,39 @@
+class Fstrm < Formula
+  desc "Frame Streams implementation in C"
+  homepage "https://github.com/farsightsec/fstrm"
+
+  stable do
```

Comment:
Don't need a `stable do` block unless you're doing something special in it, like a `stable`-only resource or dependency.